content/this-month/2023-06/index.md

```diff
@@ -87,7 +87,7 @@ After a lot of back and forth, we decided to [provide both options](https://gith
 Note that there is also some ongoing discussion [about a potential `VolatileCell` type](https://github.com/rust-osdev/volatile/issues/31) to wrap values in-place. Most implementations of such a type would require support from the Rust compiler, which [needs an RFC](https://github.com/rust-lang/unsafe-code-guidelines/issues/411). However, there is one [promising design based on zero-sized types and proc-macros](https://github.com/rust-osdev/volatile/issues/31#issuecomment-1606044353) by [@Freax13](https://github.com/Freax13) that should not require any new language features. We will continue to investigate.
 
 ### [`multiboot2`](https://github.com/rust-osdev/multiboot2)
-<span class="maintainers">Maintained by [@IsaacWoods](https://github.com/IsaacWoods), [@phip1611](https://github.com/phip1611), [@robert-w-gries](https://github.com/robert-w-gries), and [@ahmedcharles](https://github.com/ahmedcharles)</span>
+<span class="maintainers">Maintained by [@phip1611](https://github.com/phip1611)</span>
```

Comment:
Do not get me wrong; It's not like I want to take all the credits alone. But in fact I'm the only maintainer. Isaac told me a few months ago that he's very busy. The other mantainers have been inactive for many months.

Reply:
Ah sorry, I thought this was a mistake, given that there are still 4 people on the @rust-osdev/multiboot2 maintainer team. We should probably adjust that.

Reply:
@IsaacWoods Thanks for the update! Reviewing the team was just a suggestion, it's up to @phip1611 to decide. Good luck for your exam!

Reply:
I don't mind having inactive maintainers in the list. The only risk for inactive maintainers is that one day their GitHub account might be hijacked unnoticed

Reply:
Ok, then let's keep everything as it is :).